### ISSUE TITLE
It support Chinese?

### ISSUE THREAD

**Issue by jetBn** [HosseinShabani/react-native-modern-datepicker#33]
I think user can provide config to show header text as same you project jalaaliConfigs or gregorianConfigs

**Comment by HosseinShabani** [comment]
@jetBn Yes You can fork the project and make new config and use that instead of gregorianConfigs, and change your font
